### PR TITLE
Add schema and example of SQL query

### PR DESCRIPTION
**Pull request by H1mansh0**

### CODE REVIEW

File: DepartmentTopThreeSalaries.spsql
Comment (maksymloktionov) [inline DepartmentTopThreeSalaries.spsql:1]: Can we make a folder named `_sql` for these 2 files? Also, please make aliases more explicit for readability.

File: schema.png
Comment (maksymloktionov) [inline schema.png:1]: The tables lacking some metadata columns: `updated_at`, `crated_at`

Comment (maksymloktionov) [inline schema.png:1]: Can we expand dates to be either timestamps(e.g. UTC timezone?) or make them epoch int values?

**Comment by H1mansh0** [comment]
Changes applied

File: _sql/DepartmentTopThreeSalaries.spsql
Comment (maksymloktionov) [inline _sql/DepartmentTopThreeSalaries.spsql:1]: This should be `.sql` format

**Comment by maksymloktionov** [comment]
Could we remove the duplicated files outside the `_sql` directory?

File: .sql/DepartmentTopThreeSalaries.spsql
Comment (maksymloktionov) [inline .sql/DepartmentTopThreeSalaries.spsql:1]: The format for the file should be` .sql`, so basically it should have path `_sql/DepartmentTopThreeSalaries.sql`

Comment (maksymloktionov) [reply .sql/DepartmentTopThreeSalaries.spsql:1]: And we need to change the directory from `.sql `to `_sql` back

File: _sql/DepartmentTopThreeSalaries.sql [hunk 1]
```diff
@@ -0,0 +1,16 @@
+-- Department Top Three Salaries (https://leetcode.com/problems/department-top-three-salaries/description/)
+
+SELECT dprt.name AS Department, empl.name AS employee, s.salary
+FROM Employee AS empl
+INNER JOIN Department AS dprt
+ON empl.departmentId = dprt.id
+INNER JOIN (
```
[the remark this answers is not in the document]
Comment (H1mansh0) [reply _sql/DepartmentTopThreeSalaries.sql:7]: I think it's possible, but I need time to remember the context of the problem. I will push a new version in the next few days

**Comment by H1mansh0** [comment]
*SAME NAME AND SLR

File: _sql/DepartmentTopThreeSalaries.sql [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    SELECT dprt.name AS Department,
+           empl.name as Employee,
+           empl.salary AS Salary,
+           DENSE_RANK() OVER(
+            PARTITION BY departmentId ORDER BY salary DESC
+            ) AS rank
+    FROM Employee as empl
+    INNER JOIN Department as dprt
+    ON empl.departmentId = dprt.id
+    ORDER BY Department, Salary DESC
```
Comment (maksymloktionov) [inline _sql/DepartmentTopThreeSalaries.sql:13]: this is unnecessary but fine for now

**Review by maksymloktionov — APPROVED** [review]

**Pull request merged**